app/views/shelters/index.html.erb

```diff
@@ -1,4 +1,5 @@
 <h1>All Shelters</h1>
 <% @shelters.each do |shelter| %>
-  <%= shelter %>
+  <p><%= shelter.name %></p>
```

Comment:
Using ERB to display the name of each Shelter object in the database. We are able to access these because the controller passed this view an instance variable with an array of all the Shelters.

Reply:
I also would like to make the names of each shelter a link to their show page.